src/components/geo-map/geo-map.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, ElementRef, Input, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { GeoMapD3, RenderOptions } from '../../d3/geo-map.d3';
+import { A11yPlaceholderDirective } from '../../directives/a11y-placeholder/a11y-placeholder.directive';
+import { GeoMapMeta } from '../../datasets/metas/geo-map.meta';
+
+@Component({
+  selector: 'app-geo-map',
+  templateUrl: './geo-map.component.html',
+  styleUrls: ['./geo-map.component.scss'],
+})
+export class GeoMapComponent implements RenderOptions, OnInit, OnDestroy {
+  @ViewChild(A11yPlaceholderDirective, { static: true }) a11yPlaceholder: A11yPlaceholderDirective<GeoMapComponent>;
+
+  @Input() meta: GeoMapMeta;
+  @Input() height = 500;
+  @Input() width = 800;
+  @Input() topoJsonUrl = 'https://cdn.jsdelivr.net/npm/world-atlas@2/countries-10m.json';
```

Comment:
Is this ever going to be changed via input? seems like it could just be a const

Reply:
Currently not, but I was thinking of putting it in a dataset preference to choose among `countries-10m.json`, `countries-50m.json`, and `countries-110m.json`, etc. in the future.